components/x-privacy-manager/package.json

```diff
@@ -27,15 +27,15 @@
   "dependencies": {
     "@financial-times/x-engine": "file:../../packages/x-engine",
     "@financial-times/x-interaction": "file:../x-interaction",
-    "classnames": "^2.2.6"
+    "classnames": "2.3.1"
   },
   "devDependencies": {
     "@financial-times/x-rollup": "file:../../packages/x-rollup",
     "@financial-times/x-test-utils": "file:../../packages/x-test-utils",
-    "bower": "^1.8.8",
-    "check-engine": "^1.10.1",
-    "fetch-mock-jest": "^1.3.0",
-    "sass": "^1.26.5"
+    "bower": "1.8.13",
+    "check-engine": "1.10.1",
+    "fetch-mock-jest": "1.5.1",
+    "sass": "1.44.0"
```

Comment:
**question**:  were these pinned deliberately? our new Renovate config doesn't pin things any more, so i think it'll be annoyed about this

Reply:
Not exactly deliberately: this made me realise that my ~/.npmrc file has `save-exact=true`